validmind/unit_metrics/sklearn/classification/Accuracy.py

```diff
@@ -0,0 +1,63 @@
+# Copyright © 2023-2024 ValidMind Inc. All rights reserved.
+# See the LICENSE file in the root of this repository for details.
+# SPDX-License-Identifier: AGPL-3.0 AND ValidMind Commercial
+
+from dataclasses import dataclass
+from typing import Optional
+
+from sklearn.metrics import accuracy_score
+
+from validmind.vm_models import UnitMetric
+
+
+@dataclass
+class Accuracy(UnitMetric):
+    def __init__(
+        self,
+        inputs=None,
+        params=None,
+        test_id="Accuracy Score",
+    ):
+        super().__init__(test_id=test_id)
+        self.metric_inputs = inputs
+        self.params = params if params is not None else {}
+        self.type = "evaluation"
+        self.scope = "model_accuracy"
+
+    def run(self):
+        y_pred, y_true = self.get_required_inputs()
```

Comment:
Opinion: no need to have a dedicated method for this since it's only 1 extra line of code. If anything it adds complexity to the class:

```suggestion
        y_pred = self._get_y_pred()
        y_true = self._get_y_true()
```